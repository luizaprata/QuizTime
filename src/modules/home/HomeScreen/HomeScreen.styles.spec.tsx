import React from 'react'
import { render } from '@testing-library/react-native'
import {
  Container,
} from './HomeScreen.styles'

describe('HomeScreen.styles', () => {
  test('THEN Container SHOULD match with snapshot', () => {
    const container = render(<Container />).asJSON()
    expect(container).toMatchSnapshot()
  })
  
})
