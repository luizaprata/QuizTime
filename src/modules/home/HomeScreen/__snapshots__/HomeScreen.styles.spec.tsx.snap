// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeScreen.styles THEN CardsContainer SHOULD match with snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flexDirection": "row",
        },
      ]
    }
  />
</View>
`;

exports[`HomeScreen.styles THEN CardsScrollContainer SHOULD match with snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "overflow": "visible",
          "paddingBottom": 0,
          "paddingLeft": 0,
          "paddingRight": 0,
          "paddingTop": 16,
        },
      ]
    }
  />
</View>
`;

exports[`HomeScreen.styles THEN Container SHOULD match with snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <SafeAreaView
    style={
      Array [
        Object {
          "alignItems": "center",
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
          "justifyContent": "space-between",
          "paddingBottom": 0,
          "paddingLeft": 0,
          "paddingRight": 0,
          "paddingTop": 0,
        },
      ]
    }
  />
</View>
`;

exports[`HomeScreen.styles THEN ErrorMessageContainer SHOULD match with snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <View
    style={
      Array [
        Object {
          "flexBasis": 0,
          "flexGrow": 1,
          "flexShrink": 1,
        },
      ]
    }
  />
</View>
`;

exports[`HomeScreen.styles THEN RefreshContainer SHOULD match with snapshot 1`] = `
<View
  collapsable={true}
  pointerEvents="box-none"
  style={
    Object {
      "flex": 1,
    }
  }
>
  <RCTScrollView
    contentContainerStyle={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "space-between",
        "padding": 0,
      }
    }
    style={
      Array [
        Object {},
      ]
    }
  >
    <View />
  </RCTScrollView>
</View>
`;
